solution/deps.yaml

```diff
@@ -92,9 +91,9 @@ vault:
   tag: 8.7.4
   envsubst: VAULT_TAG
 zenko-operator:
-  sourceRegistry: registry.scality.com/zenko-operator
+  sourceRegistry: registry.scality.com/sf-eng
   image: zenko-operator
-  tag: 1.5.13
+  tag: 72390ca1afbf1aaf71b4017495b5d53b14fff33e
```

Comment:
will change here post release zkop

Reply:
thanks for letting me know the concern about zkop in zenko 2.6, if don't bump zkop, can we introduce[ this env](https://github.com/scality/zenko-operator/pull/373/files#diff-e6542f41494ebe04c99b56959487c4a3fcec511efddb016773806f2da7896b66R143) for pensieve-api in Zenko instead of zkop?

Reply:
I don't think so... Let's wait for a confirmation first, as we'll need a solution anyway to unblock this branch... Maybe it is fine for this bump, but we need confirmation :smile:

Reply:
ZKOP cannot be bumped in zenko 2.6 (yet), we have a ticket for this : https://scality.atlassian.net/browse/ZKOP-248
But it will probably be done next week.

Generally, if possible i think it is best to wait until CNES release (e.g. artesca 1.7.5) is complete to merge anything in zenko 2.6, to avoid adding more stuff to that release while in RC.